showcase-v1alpha2/src/v1alpha2/echo_client.ts

```diff
@@ -0,0 +1,688 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// TODO: streaming methods return untyped EventEmitter.
+// We cannot make stream.on() stronly typed because we return different types on
+// different events (e.g. stream.on('data') vs stream.on('metadata')). It may be
+// possible to list all valid types for on().
+
+import * as gax from 'google-gax';
+const gapicConfig = require('./echo_client_config') as gax.ClientConfig;
```

Comment:
Switch to `import`

Reply:
`./echo_client_config` is a plain JSON file with some configuration, so I `require` it.  I can _probably_ change that to a real TypeScript module though.